region/maridia/inner-yellow.json

```diff
@@ -252,7 +252,20 @@
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
-                    "canTrickyUseFrozenEnemies"
+                    "canTrickyUseFrozenEnemies",
```

Comment:
I feel like this is pretty trivial and am hesitating having it in hard difficulty (which it is now in, after converting it to `canTrickyUseFrozenEnemies`). There are several pretty easy places to stay safe from the fish (e.g. the door or by the item), which I think are easy enough they should just be assumed with `canSuitlessMaridia`. My preference for how this strat should be done is like this:

`canUseFrozenEnemies` plus one of the following:
hijump, gravity, spring ball midair, careful jump, canTrickyUseFrozenEnemies

and then a note saying that it can be done with two fish, or just the top fish. The top fish frozen in the right spot can be enough with a careful jump to the ledge, but it can be frozen a little higher and then rub against the fish to get onto it and then getting to the ledge is easier

Reply:
Your right but, I think its more likely someone jumps into the room without thinking then takes a hit.  Or doesn't know the door is safe.  At this level. But that pretty much is a different problem from how we've been modelling rooms.
`canUseFrozenEnemies` here wasn't liked at intermediate, but that plus a nice item or tech could be ok.  
I'm starting to think `canCarefulJump` should be moved to Hard.

Reply:
I think that navigating this room should be in logic in Intermediate and am curious where the concern is coming from. It seems easy to avoid the fish. I don't think having to use two frozen enemies is enough to make this require canTrickyUseFrozenEnemies; climbing frozen rippers in Red Tower also involves multiple enemies but that doesn't make it "tricky". Like all water rooms, there's some awkwardness with the movement, but that seems covered by requiring canSuitlessMaridia (or whatever we rename that to).

Reply:
Its hard to find a reason to keep this out of intermediate if you play safe.  The way I've done rooms till now, the fish climb is just slightly harder than canUseFrozenEnemies because you gotta position it and fish hug climb, as free actions.  

Frozen Fish Hug should be mentioned in a tech somewhere.  A new low level tech, or it could fit in carefulJump.  You get the most out of it underwater, but it can be done using any enemy that doesn't damage you.  Like ice beam snake room in speedruns use the fune for a little boost to skip a walljump.